### PR TITLE
docs: change request reject

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Updating docs about change request rejection

### CODE REVIEW

File: website/docs/reference/change-requests.md [hunk 1]
```diff
@@ -52,10 +52,11 @@ The flow can be summarized as follows:
...
 
 * **Draft** - The change request is in draft mode, and can be edited by the user that created the draft.
-* **In review** - The change request is in review mode, and can be edited by the user that created the draft. If editing ocurrs, all current approvals are revoked
+* **In review** - The change request is in review mode, and can be edited by the user that created the draft. If editing occurs, all current approvals are revoked
 * **Approved** - The change request has been approved by the required number of users.
 * **Applied** - The change request has been applied to the environment. The feature toggle configuration is updated.
-* **Cancelled** - The change request has been cancelled by the user/admin.
+* **Cancelled** - The change request has been cancelled by the change request author or admin.
+* **Rejected** - The change request has been rejected by the reviewer or admin.
```
Comment (thomasheartman) [inline website/docs/reference/change-requests.md:59]:
```suggestion
* **Rejected** - The change request has been rejected by the reviewer or by an admin. 
```

File: website/docs/reference/change-requests.md [hunk 2]
```diff
@@ -52,10 +52,11 @@ The flow can be summarized as follows:
...
 
 * **Draft** - The change request is in draft mode, and can be edited by the user that created the draft.
-* **In review** - The change request is in review mode, and can be edited by the user that created the draft. If editing ocurrs, all current approvals are revoked
+* **In review** - The change request is in review mode, and can be edited by the user that created the draft. If editing occurs, all current approvals are revoked
 * **Approved** - The change request has been approved by the required number of users.
 * **Applied** - The change request has been applied to the environment. The feature toggle configuration is updated.
-* **Cancelled** - The change request has been cancelled by the user/admin.
+* **Cancelled** - The change request has been cancelled by the change request author or admin.
```
Comment (thomasheartman) [inline website/docs/reference/change-requests.md:58]:
```suggestion
* **Cancelled** - The change request has been cancelled by the change request author or by an admin.
```

File: website/docs/reference/change-requests.md [hunk 3]
```diff
@@ -71,7 +72,7 @@ From here, if you have the correct permissions, you can approve and apply the ch
 
 Change requests have their own set of environment-specific permissions that can be applied to [custom project roles](rbac.md#custom-project-roles). These permissions let users
 
-- approve change requests
+- approve/reject change requests
```
Comment (thomasheartman) [inline website/docs/reference/change-requests.md:75]: Are these different permissions? If so, we might want to split them up into two lines here, like this:

```suggestion
- approve change requests
- reject change requests
```

**Review by thomasheartman — APPROVED** [review]
Looks good 😄  I have a few small suggestions inline. Should we write a paragraph or a few more sentences on what it means to reject a change request, or is that not necessary?

File: website/docs/reference/change-requests.md (hunk 3 above)
Comment (kwasniew) [reply website/docs/reference/change-requests.md:75]: It's the same permission and it's updated to Approve/Reject change requests in DB

**Comment by kwasniew** [comment]
@thomasheartman we'll be also updating the change request flow diagram which may suffice. I think it's quite obvious what reject is, I only tried to stress a difference between cancel you changes and reject someone else's.

**Comment by thomasheartman** [comment]
Sounds good! I'll trust your judgment there. We can always update it if we get questions later.

**Pull request merged**